descwl_shear_sims/shifts.py

```diff
@@ -197,7 +215,7 @@ def get_grid_shifts(*, rng, dim, buff, spacing):
 
 def get_random_shifts(*, rng, dim, buff, size):
     """
-    get a set of gridded shifts, with random shifts at the pixel scale
+    get a set of gridded shifts in a square, with random shifts at the pixel scale
```

Comment:
Actually the original doc string was quite wrong, this is  not gridded.

It should be something like "get a set of random shifts in a square"

Reply:
Done!!